scripts/parse_tools/xml_tools.py

```diff
@@ -51,6 +51,13 @@ def call_command(commands, logger, silent=False):
     False
     >>> call_command(['ls'], _LOGGER)
     True
+    >>> try:
+    ...    call_command(['ls','--invalid-option'], _LOGGER)
+    ... except CCPPError as e:
+    ...    print(str(e))
+    Execution of 'ls --invalid-option' failed with code:
+    Error output: ls: unrecognized option '--invalid-option'
+    Try 'ls --help' for more information.
```

Comment:
Suggest a new test for the dual output case:
```suggestion
    Try 'ls --help' for more information.
    >>> try:
    ...    os.chdir(os.path.dirname(__file__))
    ...    call_command(['ls', os.path.basename(__file__), 'foo.bar.baz'], _LOGGER)
    ... except CCPPError as e:
    ...    print(str(e))
    Execution of 'ls xml_tools.py foo.bar.baz' failed with code: 2
    xml_tools.py
    Error output: ls: cannot access 'foo.bar.baz': No such file or directory
```

Reply:
thanks for the new doctest! added.